source/developers-guide/shopware-5-upgrade-guide-for-developers/index.md

```diff
@@ -17,6 +17,673 @@ including minor and bugfix releases, refer to the `UPGRADE.md` file found in you
 
 <div class="toc-list"></div>
 
+## Shopware 5.3
+
+### System requirements changes
+
+The minimum PHP version is still **PHP 5.6.4 or higher**.
+
+### Smarty Rendering
+
+#### Form Fields
+
+Only variables that were previously assigned to the view are rendered. In addition, smarty function calls are no longer executed.
```

Comment:
Smarty functions in form templates have been disabled.  Also no new variables can be added to the template.